juju-tutorials/juju-tricks.md

```diff
@@ -0,0 +1,260 @@
+[comment]: <> (Juju Tricks)
+
+## Introduction
+
+Duration: 0:01
+
+This article is compilation of CLI tricks, useful for recovery of juju deployments after OS services' hosts reboot / crash / network splits / etc.
+
+### Requirements
+
+There is no specific setup precedure for this tutorial because it contains commands that are applicable in various situations in any juju deployment. Most of the commands are using just standard cli utilities like `grep`, `sed` or `awk`.
```

Comment:
I suggest to use "Juju" as the product starts with upper case. On the previous paragraph, "CLI" was written in upper case, while in this one it is written in lower case ("cli"). Since it's an acronym, I suggest to use the upper case version.